features/k8s-gateway-apis.md

```diff
@@ -141,25 +141,34 @@ N/A: API is not owned by Istio. But its been extensively reviewed in the Kuberne
 
 **Design**
 
-- [ ] Design doc describing the intention of the feature, how it will be
+- [x] Design doc describing the intention of the feature, how it will be
 	implemented, and any thoughts on how to test the feature has been approved by
 	relevant work group leads
-- [ ] Feature coverage and test plans written and approved.
+
+	API design is by Kubernetes, we are only an implementation. The implementation has been reviewed and approved.
+
+- [x] Feature coverage and test plans written and approved.
+
+	Most testing is handled by the upstream conformance tests. Special tests for Istio specific features exist as end to end tests.
```

Comment:
Not a blocking comment, but can we get a link to these tests?  I'd like to understand how they work.

Reply:
https://github.com/kubernetes-sigs/gateway-api/tree/main/conformance